packages/rrweb/test/__snapshots__/integration.test.ts.snap

```diff
@@ -13032,6 +13032,21 @@ exports[`record integration tests should not record blocked elements dynamically
         {
           \\"parentId\\": 16,
           \\"nextId\\": 18,
+          \\"node\\": {
+            \\"type\\": 2,
+            \\"tagName\\": \\"iframe\\",
+            \\"attributes\\": {
+              \\"class\\": \\"rr-block\\",
+              \\"rr_width\\": \\"104px\\",
+              \\"rr_height\\": \\"104px\\"
+            },
+            \\"childNodes\\": [],
+            \\"id\\": 23
+          }
+        },
```

Comment:
This is correct as it records the dimensions, but omits the src since it is blocked.